### DISCUSSION TITLE
Add something about accessibility of ABACUS repo on Gitee into the docs

### DISCUSSION THREAD

**Discussion by scott-5** [deepmodeling/abacus-develop#5605] [Ideas]
I noticed that this repo mirrors self to Gitee. And real-time synchronization is truly achieved now. Despite easy_install part in the docs was already hints that code could be cloned from Gitee, but I remember a long time ago, the repo on Gitee was outdated. So some old users may still think it's outdated.

**Comment by QuantumMisaka** [comment]
Good advice. I also notice it, I do think the update of docs is needed

Besides, I wonder the mechanism for synchronization, may the dependence package of toolchain can also be synchronized in gitee

**Reply by scott-5** [reply]
I think it is still effective. Any push (including toolchain) will be synchronized in real-time.

**Reply by QuantumMisaka** [reply]
I do not mean the toolchain source-code in ABACUS repo, but the dependencies which toolchain will download. They may can be set as an image source in gitee.

Back to main topic: docs is truly needed to be updated (although lots of expertises do not like gitee, but it is usable.)

**Reply by QuantumMisaka** [reply]
#5666

**Reply by scott-5** [reply]
Got it. It's best if the dependencies already has an image on Gitee, otherwise it may be quite troublesome.
